src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -63,6 +63,48 @@ internal virtual void ToVector4(ReadOnlySpan<TPixel> sourceColors, Span<Vector4>
             }
         }
 
+        /// <summary>
+        /// Bulk version of <see cref="IPixel.PackFromArgb32(Argb32)"/>
+        /// </summary>
+        /// <param name="sourceVectors">The <see cref="Span{T}"/> to the source vectors.</param>
+        /// <param name="destinationColors">The <see cref="Span{T}"/> to the destination colors.</param>
+        /// <param name="count">The number of pixels to convert.</param>
+        internal virtual void PackFromArgb32(ReadOnlySpan<Argb32> sourceVectors, Span<TPixel> destinationColors, int count)
```

Comment:
These should be generated via `PixelOperations{TPixel}.Generated.tt`

Reply:
I've moved the changes to the .tt file.